spec/dummy/test/components/stories/demo/heading_component_stories.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Demo
+  class HeadingComponentStories < ViewComponent::Storybook::Stories
+    stories_title 'Heading Component'
```

Comment:
Isn't this exactly the value we'd get without this additional feature?

Reply:
In this particular case the title would resolve to `Demo/Heading Component` - see my full comment for more on why this isn't always desirable (in our case)